File: 1. Conditional Problems/Conditional Problems in Python/p19.py
# // write a program to check whether the number is a perfect square number or not

import math

# Input the number
n = int(input("Enter n: "))

# Calculate the integer square root
sqr = int(math.sqrt(n))

# Check if the square of the integer square root equals the original number
if sqr * sqr == n:
    print(f"{n} is a Perfect square Number")
else:
    print(f"{n} is not a Perfect square Number")

File: 1. Conditional Problems/Condional Problems in C/p19.c
// write a program to check whether the number is a perfect square number or not

#include <stdio.h>
#include <math.h>

int main()
{
    int n, sqr;
    printf("Enter n: ");
    scanf("%d", &n);

    sqr = (int)sqrt(n);

    if (sqr * sqr == n)
    {
        printf("%d is a Perfect square Number", n);
    }

    else
    {
        printf("%d is a not Perfect square Number", n);
    }
    return 0;
}
